app/views/cookies/consent.html.erb

```diff
@@ -37,9 +37,9 @@
           <%= link_to t('cookies.buttons.embedding'), cookies_consent_path(allow: 'necessary,embedding'),
                     method: :post, class: 'btn btn-default' %>
           <%= link_to t('cookies.buttons.all'), cookies_consent_path(allow: CookieConsent::OPTIONS.join(',')),
-                    method: :post, class: 'btn btn-default' if (Seek::Config.google_analytics_enabled || Seek::Config.piwik_analytics_enabled) %> 
+                    method: :post, class: 'btn btn-default' if (Seek::Config.google_analytics_enabled || Seek::Config.piwik_analytics_enabled || Seek::Config.custom_analytics_snippet_enabled) %>
```

Comment:
I think maybe consolidate this into a new method in the config class, something like `Seek::Config.analytics_enabled`

and use it in `app/views/layouts/_cookies_banner.html.erb` as well